lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -732,6 +735,26 @@ def _get_node_information(self, node: Node) -> Dict[str, str]:
             node.log.debug("detecting vm generation...")
             information[KEY_VM_GENERATION] = node.tools[VmGeneration].get_generation()
             node.log.debug(f"vm generation: {information[KEY_VM_GENERATION]}")
+            node.log.debug("detecting mana driver enabled...")
+            information[KEY_MANA_ENABLED] = node.tools[KernelConfig].is_enabled(
+                "CONFIG_MICROSOFT_MANA"
+            )
+            node.log.debug(f"mana enabled: {information[KEY_MANA_ENABLED]}")
+
+            node.log.debug("detecting nvme driver enabled...")
+
+            _has_nvme_core = node.tools[KernelConfig].is_built_in(
```

Comment:
Should they be test cases, instead of collecting information from all cases?

Reply:
I believe we concluded to continue with this approach?

Reply:
yes